### DISCUSSION TITLE
other/notion-linebot/

### DISCUSSION THREAD

**Discussion by giscus** [Jim876633/Jim-Docusaurus#23] [Announcements]
# other/notion-linebot/

會需要串接 Line Bot 跟 Notion API，是因為我常常在 FB 或 IG 等社群軟體看到一些不錯的文章或影片，就會分享到 Line 的群組，然後有空的時候就會把它整理成 Notion 的 DataBase 來記錄哪些有看過哪些還沒，但是這樣的流程實在是太麻煩了，而且有時候還要去對哪些已經複製過了，所以就想說看能不能用 Line Bot + Notion API 來幫我做這件事情。

https://jimhuang.dev/other/notion-linebot/

**Comment by yvette-chang** [comment]
你好，想請問"建立 Server"的部分是要在哪裡輸入程式碼？謝謝你。
